addons/mass_mailing_sale/models/utm.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class UtmCampaign(models.Model):
+    _inherit = 'utm.campaign'
+
+    ab_testing_winner_selection = fields.Selection(selection_add=[
+        ('sale_quotation_count', 'Quotations'),
+        ('sale_invoiced_amount', 'Revenues'),
```

Comment:
Probably highly inefficient ... those fields were added to display a button on form view. On a real prod DB with millions of records, this could probably be a mega sprotch.